config.js

```diff
@@ -27,15 +27,8 @@ config.usersExemptFromMaximum = process.env.EXEMPT_USERS?.split(",") || [
 config.initialGoldenRecognitionHolder =
   process.env.GOLDEN_RECOGNIZE_HOLDER || "UE1QRFSSY";
 config.redemptionAdmins = process.env.REDEMPTION_ADMINS?.split(",") || [
-  "U6BS54PJM",
   "U02KPMFA9DG",
   "U04666K57CP",
```

Comment:
Who are these users?

Reply:
U02KPMFA9DG = Smith Maze
U04666K57CP = Danielle Johnson

Reply:
When redeeming an item, Smith, Danielle and Layne would get pulled into a dm message with Gratibot. Since we no longer have Layne with us I removed his id from the list.

Reply:
Sounds good, maybe just leave the `// <user>` comments beside each of the users so unfamiliar contributors know.

Reply:
I can do that now

Reply:
done